alert-engine/src/main/scala/io/phdata/pulse/alertengine/AlertEngineImpl.scala

```diff
@@ -70,23 +66,19 @@ class AlertEngineImpl(solrServer: CloudSolrServer, notificatonServices: Notifica
   def triggeredAlert(applicationName: String, alertRule: AlertRule): Option[TriggeredAlert] =
     if (AlertsDb.shouldCheck(applicationName, alertRule)) {
       try {
-        val alias = s"${applicationName}_all"
-        val query = new SolrQuery(alertRule.query)
-        query.set("fl", "*") // return full results
-        query.set("collection", alias)
-        val results    = solrServer.query(query).getResults
-        val numFound   = results.getNumFound
-        val resultsSeq = results.asScala
-        val threshold  = alertRule.resultThreshold.getOrElse(0)
+        val alias     = s"${applicationName}_all"
+        val results   = solrService.query(alias, alertRule.query)
```

Comment:
The solr libs aren't available everywhere, so the logic to do a query is now in the Pulse `SolrService`